src/css/pager.module.css

```diff
@@ -0,0 +1,22 @@
+.pager-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 80px 40%;
+}
+
+.pager-link {
+  text-decoration: none;
+  color:  #858585;
+}
+
+.disabled-pager-link {
+  text-decoration: none;
+  color:  #B6B6B6;
+  cursor: not-allowed;
+}
+
+.selected-page {
+  text-decoration: none;
+  color:  #EE1A64;
```

Comment:
maybe you can centralize all colors in a css file and import them from here, as we use in react :smile: